task_11/css/style.css

```diff
@@ -0,0 +1,719 @@
+    position: absolute;
+    left: 1.5em;
+    top: 18px;
+}
+
+
+
+/*bottom form block*/
+.content__wrapper .content__bottom{
+    margin: 1em 0 0 0;
+}
+.content__wrapper .content__bottom p:not(.title){
+    padding: 1em 0;
+    color: #9fa5b8;
+}
+.content__wrapper .content__bottom p a{
+    padding: 1em 0;
+    color: #9fa5b8;
+}
+.content__wrapper .content__bottom .flex_row-start{
+    display: flex;
+    justify-content: flex-start;
+}
+.content__wrapper .content__bottom .flex_row-end{
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin-top: 4em;
+    flex-wrap: wrap;
+}
+.content__wrapper .content__bottom .flex_row-end label[for="agree"] a{
+    color: #6e768f;
+}
+.content__wrapper .content__bottom .flex_row-end .button__green{
+    margin: 0 1em;
+}
+@media screen and (max-width: 760px),(min-resolution:270dpi){
+    .content__wrapper .content__bottom .flex_row-end{
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+    }
+    .content__wrapper .content__bottom .flex_row-end .button__green{
+        margin: 1em 0;
+    }
+}
+.content__wrapper .content__bottom .flex_row-start .title{
+    display: block;
+    font-size: 85%;
+    margin-left: 0.5em;
+    text-transform: uppercase;
+    font-weight: 700;
+    color: #a5abc5;
+}
+.content__wrapper .content__bottom .flex_row-start div{
+    margin: 0 1em 0 0;
+}
+.content__wrapper .content__bottom .flex_row-start input:not([type="radio"]){
+    color: #6e768f;
+    border: 1px solid #d7d9e0;
+    border-radius: 3px;
+    padding: 0.5em 2.5em 0.5em 1em;
+}
+.content__wrapper .content__bottom .flex_row-start
+input:not([type="radio"])+label{
+    position: relative;
+    display: block;
+    width: 100%;
+    height: 100%;
+}
+.content__wrapper .content__bottom .flex_row-start
+input:not([type="radio"]):valid+label:before{
+    font-family: Font Awesome\ 5 Free;
+    background: #76a7e1;
+    color: #fff;
+    content: "\f058";
+    position: absolute;
+    top: -30px;
+    right: 5px;
+    font-size: 100%;
+    width: 15px;
+    padding: 0 3px;
+    border-radius: 100%;
+    text-align: center;
+}
+.content__wrapper .content__bottom .flex_row-start
+input:not([type="radio"]):invalid+label:before{
+    font-family: Font Awesome\ 5 Free;
+    background: red;
+    color: #fff;
+    content: "\f057";
+    position: absolute;
+    top: -30px;
+    right: 5px;
+    font-size: 100%;
+    width: 15px;
+    padding: 0 3px;
+    border-radius: 100%;
+    text-align: center;
+}
+.content__wrapper .content__bottom .flex_row-start
+#card+label:before{
+    font-family: Font Awesome\ 5 Free;
+    color: #d7d9e0;
+    content: "\f09d";
+    background: none;
+    position: absolute;
+    top: -32px;
+    right: 5px;
+    font-size: 110%;
+    width: 30px;
+    padding: 0 3px;
+    border-radius: 100%;
+    text-align: center;
+}
+.content__wrapper .content__bottom .flex_row-start #card{
```

Comment:
its but to not use ids in css. their specificity is to big. (Shark)
![Spec](https://i.imgur.com/O6Oob.gif)

Reply:
But remember, for each rule there is at least one exceptions.

Just in your case, `.content__wrapper .content__bottom .flex_row-start` is redundant because id __must__ be only one per page, so we can select just with `#card`

Reply:
However, think about naming of IDs. There may be any _card_, it does not look great.

Reply:
Thanks for review and advice! I'll try to pay more attention to specificity of selectors in future

Reply:
Totally forgot about weight of IDs, so for now i just replaced them with classes.